bundle-ocp/manifests/das-operator.clusterserviceversion.yaml

```diff
@@ -54,7 +54,7 @@ metadata:
     features.operators.openshift.io/token-auth-gcp: "false"
     operators.openshift.io/valid-subscription: '["OpenShift Kubernetes Engine", "OpenShift
       Container Platform", "OpenShift Platform Plus"]'
-    operators.operatorframework.io/builder: operator-sdk-v1.41.1
```

Comment:
we should probably use the same version or higher?

Reply:
auto generated